test/integration/prepare_int_test.go

```diff
@@ -47,6 +50,38 @@ func (suite *PrepareIntegrationTestSuite) TestPrepare() {
 		return
 	}
 	suite.AssertConfig(filepath.Join(ts.Dirs.Cache, "bin"))
+
+	// Verify autostart was enabled.
+	cfg, err := config.New()
+	suite.Require().NoError(err)
+	as, err := autostart.New(svcAutostart.App, ts.SvcExe, nil, svcAutostart.Options, cfg)
+	suite.Require().NoError(err)
+	enabled, err := as.IsEnabled()
+	suite.Require().NoError(err)
+	suite.Assert().True(enabled, "autostart is not enabled")
+
+	// When installed in a non-desktop environment (i.e. on a server), verify the user's ~/.profile was amended.
+	if runtime.GOOS == "linux" {
+		homeDir, err := homedir.Dir()
+		suite.Require().NoError(err)
+		profile := filepath.Join(homeDir, ".profile")
+		suite.Contains(string(fileutils.ReadFileUnsafe(profile)), as.Exec, "autostart should be configured for Linux server environment")
```

Comment:
nit: Store this string in a variable and reuse.

Reply:
Okay.